kube_hunter/modules/report/ndjson.py

```diff
@@ -0,0 +1,52 @@
+import json
+from kube_hunter.modules.report.base import BaseReporter
+
+
+class NDJSONReporter(BaseReporter):
+    def get_report(self, **kwargs):
+        report = super().get_report(**kwargs)
+
+        nodes = report["nodes"]
+        services = report["services"]
+        vulnerabilities = report["vulnerabilities"]
+
+        flattenedEntries = []
+        for node in nodes:
+            nodeAdded = False
```

Comment:
What is the motivation here?
Don't we want to print an entry with node details always?
Is there a duplicates problem with consumers of this output?